File: src/zelpage/razeni/TrainView.java
package zelpage.razeni;

import android.app.Activity;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;

public class TrainView extends Activity {
	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.trainview);
		
		Intent i = this.getIntent();
		Uri data = i.getData();
		
		String trainnumber = data.getHost();
		
		
	}
}

File: app/src/main/java/zelpage/razeni/TrainView.kt
package zelpage.razeni

import android.app.Activity
import android.net.Uri
import android.os.Bundle

class TrainView : Activity() {
	public override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.trainview)

		val data : Uri = this.intent.data

		val trainnumber = data.host
	}
}
